docs/support.rst

```diff
@@ -26,12 +26,10 @@ Stay connected.
 Mailing list
 ------------
 
-New Mailing List:
-
-`borgbackup@python.org <https://mail.python.org/mailman/listinfo/borgbackup>`_
-
-Just read that page to find out about the mailing list, its topic, how to subscribe,
-how to unsubscribe and where you can find the archives of the list.
+To find out about the mailing list, its topic, how to subscribe, how to
```

Comment:
We have a mailing list!

Homepage:
<https://mail.python.org/mailman/listinfo/borgbackup>

Here you can read the archives and subscribe.